cmd/fpga_admissionwebhook/fpga_admissionwebhook.go

```diff
@@ -62,6 +77,57 @@ func addToScheme(scheme *runtime.Scheme) {
 	admissionregistrationv1beta1.AddToScheme(scheme)
 }
 
+// TODO: get rid of hardcoded translations of FPGA resource names to region interface IDs
+func region2interfaceID(regionName string) string {
+	switch regionName {
+	case "arria10":
+		return "ce48969398f05f33946d560708be108a"
+	}
+
+	return ""
+}
+
+// TODO: get rid of hardcoded translations of FPGA resource names to region interface IDs
+func af2afuID(afName string) string {
+	switch afName {
+	case "nlb0":
+		return "d8424dc4a4a3c413f89e433683f9040b"
+	case "nlb3":
+		return "f7df405cbd7acf7222f144b0b93acd18"
+	}
+
+	return ""
+}
+
```

Comment:
would it be more "go way" if this function returns string, error?

Reply:
+1. and only once specify error message here instead of all places where it is used.

Reply:
done